README.md

```diff
@@ -90,19 +90,24 @@ You can use strings instead: `'QuerySet[MyModel]'` and `'Manager[MyModel]'`, thi
 
 Currently we [are working](https://github.com/django/django/pull/12405) on providing `__class_getitem__` to the classes where we need them.
 
-### How can I use HttpRequest with custom user model?
+### How can I create a HttpRequest that's guaranteed to have an authenticated user?
```

Comment:
The previous content here wasn't up to date, the functionality described (using a custom user model) just works out of the box with the vanilla `HttpRequest` class the way django-stubs currently works.

New content describes a similar use case though.

Reply:
👍